src/org/zaproxy/zap/extension/pscanrulesAlpha/CacheableScanner.java

```diff
@@ -467,9 +468,19 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 							} else {
 								//we have one expiry, and one date. Yippee.. Are they valid tough??
 								//both dates can be invalid, or have one of 3 formats, all of which MUST be supported!
-								Date expiresDate = DateUtil.parseDate( expiresHeader );
+								Date expiresDate = null;
+								try {
+									expiresDate = DateUtil.parseDate( expiresHeader );
+								} catch(DateParseException dpe) {
+									// There was an error parsing the date, leave the var null
+								}
+								Date dateDate = null;
```

Comment:
Was the variable moved outside the if on purpose?

Reply:
Nope, probably a bad copy/paste.